solr/contrib/ltr/src/java/org/apache/solr/ltr/store/rest/ManagedModelStore.java

```diff
@@ -110,7 +115,7 @@ public void loadStoredModels() {
       final List<Map<String,Object>> up = (List<Map<String,Object>>) managedData;
```

Comment:
Yes, once this PR has made models pluggable we have plans (another ticket/pull request) to revisit ManagedModelStore overall including variable naming as you mention here.